ts-sdk/whirlpool/tests/config.test.ts

```diff
@@ -43,6 +45,18 @@ describe("Configuration", () => {
     );
   });
 
+  it("Should be able to set the network", async () => {
```

Comment:
Nit: should be able to set whirlpools config based on network

You're not actually setting the network